pyccel/ast/core.py

```diff
@@ -1915,6 +1916,20 @@ def has_default(self):
         (if not then it must be provided)
         """
         return self._value is not None
+    
+    @property
+    def is_const(self):
+        """ Indicates whether the argument has been modified
+        (if not then it can be declared as a constant)
+        """
+        return self._is_const
```

Comment:
I thought this information was already in the variable. Don't you just need to do `self._var.is_const`?